src/Pages/Channels.razor

```diff
@@ -105,6 +105,7 @@
                             {
                                 <SelectItem Value="@(wallet.Id)">@wallet.Name</SelectItem>
                             }
+                              <SelectItem Value="@(_wallets.Count +1)">Unknown</SelectItem>
```

Comment:
I recommend using -1 for this. -1 It's never going a wallet id, so there will never be a mistake

Reply:
With this fix we will get unexpected behaviour 

https://github.com/Elenpay/NodeGuard/assets/81957232/149d0a96-b29b-47bf-a9f2-66906ebce8fe

Reply:
```suggestion
                              <SelectItem Value= "-1">Unknown</SelectItem>
```

Reply:
Ok. I've fixed according to the comments.